tests/playbooks/tests_bond_cloned_mac.yml

```diff
@@ -69,6 +70,31 @@
               - item.stdout.endswith(item.item.mac)
             msg: cloned-mac-address is configured incorrectly
           loop: "{{ cloned_mac_address.results }}"
+          when: network_provider == 'nm'
```

Comment:
The verification scheme `nmcli -f 802-3-ethernet.cloned-mac-address con show` did not work for initscripts provider, thus I verify the MAC address in the `ip addr show` for initscripts provider.

Reply:
ok thanks a lot for the work